README.md

```diff
@@ -84,6 +84,17 @@ Delete cookie:
 Cookies.remove('name');
 ```
 
+Please notice that removing unexisting cookie does not raise any exception nor return any value
+
+```javascript
+var whatWasTheValue = Cookies.remove('nothing');
+if (whatWasTheValue === undefined) {
+  console.log("I will be excuted");
+} else {
+  console.log("I will never be executed");
```

Comment:
I mainly started this `#remove` research because comment here:
```
Cookies.remove('name'); // fail!
```
didn't make me sure enough if fail means exception, undefined or something else. 

In Ruby `fails` means throw exception, so I needed to double check this one 😆